### DISCUSSION TITLE
请问vscode中正向、方向查找的快捷键是什么？

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#143] [Q&A] [answered]
请问vscode中正向、方向查找的快捷键是什么？谢谢！
就是点击代码跳转到pdf位置，点击pdf位置跳转到代码。

**Comment by zepinglee** [comment, marked as answer]
- <https://github.com/James-Yu/LaTeX-Workshop/wiki/View#synctex> 
- <https://github.com/James-Yu/LaTeX-Workshop/wiki/View#latex-workshopviewpdfinternalsynctexkeybinding>

**Reply by earthmoon** [reply]
> * https://github.com/James-Yu/LaTeX-Workshop/wiki/View#synctex
> * https://github.com/James-Yu/LaTeX-Workshop/wiki/View#latex-workshopviewpdfinternalsynctexkeybinding
反向：Ctrl+左键，正向：Ctrl+Alt+j   好像是这样。谢谢
